tests/core/test_training.py

```diff
@@ -56,25 +58,19 @@ async def test_story_visualization_with_merging(default_domain: Domain):
     assert 20 < len(generated_graph.edges()) < 33
 
 
-async def test_training_script(tmp_path: Path):
-    await train(
-        DEFAULT_DOMAIN_PATH_WITH_SLOTS,
-        DEFAULT_STORIES_FILE,
-        str(tmp_path),
-        policy_config="data/test_config/max_hist_config.yml",
-        interpreter=RegexInterpreter(),
-        additional_arguments={},
-    )
-    assert True
+async def test_training_script_without_max_history_set(
+    tmp_path: Path, monkeypatch: MonkeyPatch
+):
+    tmpdir = str(tmp_path)
 
+    policy_train = Mock()
+    monkeypatch.setattr(TEDPolicy, "train", policy_train)
```

Comment:
why not removing `TEDPolicy` from the config file in the first place instead of mocking?